src/index.ts

```diff
@@ -17,15 +16,19 @@ async function main() {
   // TODO: Fail if any required values missing from .env
   const stakingPoolPublicKey: string = process.env.POOL_PUBLIC_KEY || "";
   const globalSlotStart = Number(process.env.GLOBAL_SLOT_START) || 0;
-  const minimumHeight = Number(process.env.MIN_HEIGHT); // This can be the last known payout or this could be a starting date
-  const minimumConfirmations = Number(process.env.MIN_CONFIRMATIONS) || 0;
-  const slotsPerEpoch = Number(process.env.SLOTS_PER_EPOCH);
-  const commissionRate = Number(process.env.COMMISSION_RATE);
-  const transactionFee = Number(process.env.SEND_TRANSACTION_FEE) || 0;
+  const minimumHeight = Number(process.env.MIN_HEIGHT) || 0; // This can be the last known payout or this could be a starting date
+  const minimumConfirmations = Number(process.env.MIN_CONFIRMATIONS) || 290;
+  const slotsPerEpoch = Number(process.env.SLOTS_PER_EPOCH) || 7140;
+  const commissionRate = Number(process.env.COMMISSION_RATE) || 0.05;
+  const transactionFee = (Number(process.env.SEND_TRANSACTION_FEE) || 0) * 1000000000;
   const nonce = Number(process.env.STARTING_NONCE) || 0;
-  const senderKeys: keypair = {
-    privateKey: process.env.PRIVATE_KEY || "",
-    publicKey: process.env.PUBLIC_KEY || ""
+  let generateEphemeralSenderKey = false;
+  if( typeof(process.env.SEND_EPHEMERAL_KEY) === 'string' && process.env.SEND_EPHEMERAL_KEY.toLowerCase() == 'true'){
+    generateEphemeralSenderKey = true;
+  };
```

Comment:
Should be able to make this a single assignment line.

Reply:
merging without this so we don't have this long-running; add to your branch if desired.